cosmwasm/contracts/crosschain-swaps/src/checks.rs

```diff
@@ -1,7 +1,19 @@
 use cosmwasm_std::{Addr, Deps};
 use regex::Regex;
 
-use crate::{state::CHANNEL_MAP, ContractError};
+use crate::{
+    state::{CHANNEL_MAP, CONFIG},
+    ContractError,
+};
+
+pub fn check_is_contract_owner(deps: Deps, sender: Addr) -> Result<(), ContractError> {
+    let config = CONFIG.load(deps.storage).unwrap();
+    if config.owner != sender {
```

Comment:
interesting, for some reason I thought owner was a native concept in cosmwasm, rather than just a contract variable

Reply:
There's a native contract admin, that is allowed to migrate the contract, but if you want to use it for internal stuff you need to track it (you may want the internal owner and the admin to be different!)

Reply:
Ah I see, perhaps lets call this "governor", to help distinguish?

Reply:
I have a branch changing this to "governor" but the swaprouter contract (that's already on-chain)  uses "owner".

The cosmwasm internal one is called "admin"

Reply:
Are you planning to do the governor rename as a second PR?

Reply:
I was thinking of pushing it here, just thought I'd double check if we wanted this considering that swaprouter uses owner. I feel like they should use the same name. 

I can update it on swaprouter too, but then it will be different than the one on chain 😅

Reply:
I'm a fan of governor personally. the inconsistency between the two doesn't really bother me FWIW

Reply:
ok! I'll push the governor change then

Reply:
pushed